src/lib/itemset.rs

```diff
@@ -107,7 +107,7 @@ impl Itemset {
                     dot = y;
                 }
                 None => {
-                    match zero_todos.iter_set_bits().next() {
+                    match zero_todos.iter_set_bits(..).next() {
```

Comment:
What's the difference between the two?

Reply:
Good question. It's due to a new feature in `Vob` https://github.com/softdevteam/vob/pull/4 -- the two calls above are morally equivalent, but you need to use `..` in the new version of `Vob`.

Reply:
I see.